packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -134,6 +136,13 @@ class Dismissible extends StatefulWidget {
   /// [direction] property.
   final Map<DismissDirection, double> dismissThresholds;
 
+  /// Defines the duration for card to come back to original position if not dismissed.
+  final Duration rollbackDuration;
+
+  /// Defines the end offset across the main axis after the card is dismissed.
+  /// If non-zero value is given then widget moves in cross direction depending on whether it is positive or negative.
+  final double crossAxisEndOffset;
```

Comment:
Add a blank line in the documentation after the first sentence (the first paragraph is used when we need a short description).